src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -73,6 +78,12 @@ impl Default for TimeTriggerInterval {
     }
 }
 
+#[derive(Debug, Error)]
+enum TimeTrigerError {
+    #[error("The integer value {0:?} for the specified time trigger interval is too large, it must be less than 9,223,372,036,854,775,807 seconds.")]
+    TooLargeInterval(TimeTriggerInterval),
```

Comment:
This error is too specific, just say something like "The value specified as a time trigger is out of bounds, ensure it fits in a i64: : '{0:?}'"